run-nitro-test-node/action.yml

```diff
@@ -20,6 +26,16 @@ runs:
         cd nitro
         ./test-node.bash --init --no-blockscout &
```

Comment:
I think you could use the environment variable in the run command itself, rathaer than creating a separate step.

[This](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#example-specifying-inputs) suggests it's available as an env var like `INPUT_<VARNAME>`, or there may be other ways to access it.

Reply:
Was able to do it inside a single step with a sort of ternary expression, like here: https://github.com/actions/runner/issues/409

@yahgwai